sdk/servicebus/azure-servicebus/tests/async_tests/test_sb_client_async.py

```diff
@@ -661,11 +662,11 @@ async def test_custom_endpoint_connection_verify_exception_async(self,
         hostname = f"{servicebus_namespace.name}{SERVICEBUS_ENDPOINT_SUFFIX}"
         credential = AzureNamedKeyCredential(servicebus_namespace_key_name, servicebus_namespace_primary_key)
 
-        client = ServiceBusClient(hostname, credential, connection_verify="cacert.pem", uamqp_transport=uamqp_transport)
+        certfile = certifi.where()
+        client = ServiceBusClient(hostname, credential, connection_verify=certfile, uamqp_transport=uamqp_transport)
         async with client:
-            with pytest.raises(ServiceBusError):
```

Comment:
I think this was failing on before b/c the certfile path was not correct. But it shouldn't be.

Reply:
it wasn't failing on other uamqp runs I had with settlement and it looks like it only failed on some of the os runs - could there be a change in the code that caused this? Or do you think that it was passing on accident?

Reply:
Yeah it was only failing on Linux so I'm not sure tbh. Maybe it has something to do with the cacert.pem file actually being detected with the path on other OS'es but not linux and only for uamqp?

 Going to see if the current fix works.

Reply:
added the exception for connection_verify back in